jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layout/BoxPage.java

```diff
@@ -1,11 +1,10 @@
 package io.github.com.pages.layout;
 
-import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.composite.WebPage;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
 import com.epam.jdi.light.material.elements.layout.Box;
 
-public class BoxFrame extends Section {
-
+public class BoxPage extends WebPage {
     @UI("//button[contains(@class,'MuiButton-contained')]")
     public Box containedBox;
```

Comment:
please use Button type for this variable from html module
and then we can delete Box class
same for others